compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.td

```diff
@@ -106,6 +100,8 @@ def AMDAIETileAndFuse :
   let options = [
     Option<"useSCFFor", "use-scf-for", "bool", /*default=*/"false",
       "Use scf.forall by default for the corresponding tiling level">,
+    Option<"useFusion", "use-fusion", "bool", /*default=*/"true",
```

Comment:
Maybe we can do that as a follow up, but we want to use fusion all the time on this hardware. We just need to be careful about the tile sizes used.